Backend/NoCode.FlowerShop.Application/Common/Behaviors/ValidationExtensions.cs

```diff
@@ -22,4 +23,20 @@ public static class ValidationExtensions
                 context.AddFailure("Password must contain at least one digit");
         });
     }
+    
+    public static IRuleBuilderOptionsConditions<T, string> ValidUrl<T>(this IRuleBuilder<T, string> ruleBuilder)
+    {
+        return ruleBuilder.Custom((url, context) =>
+        {
+            const string pattern 
+                = @"^[(http(s)?):\/\/(www\.)?a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)$";
+
+            var regex = new Regex(pattern);
```

Comment:
FYI: 
`.NET7` has introduced new patterns for more efficient Regex creation.

More efficient way would be to have partial regex class in the same file:
```
[GeneratedRegex("@"^[(http(s)?):\/\/(www\.)?a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)$", RegexOptions.IgnoreCase)]
private static partial Regex UrlRegex();
```